clients/python/dd_client/bench.py

```diff
@@ -0,0 +1,200 @@
+"""
```

Comment:
just curious, can we use `git mv` for renaming `dd_bench.py` into `bench.py` ?

Reply:
It's git that have record my `mv` into an `rm` / `add`. I think this is because the dd_client directory was not existing in the tree before.